source/isaaclab/isaaclab/envs/direct_marl_env.py

```diff
@@ -133,6 +133,8 @@ def __init__(self, cfg: DirectMARLEnvCfg, render_mode: str | None = None, **kwar
             print("[INFO]: Starting the simulation. This may take a few seconds. Please wait...")
             with Timer("[INFO]: Time taken for simulation start", "simulation_start"):
                 self.sim.reset()
+                # update scene to pre populate data buffers for assets and sensors
+                self.scene.update(dt=self.physics_dt)
```

Comment:
I think the code doc should define the purpose not the operation


```suggestion
                # update scene to pre populate data buffers for assets and sensors
                # this is needed for the observation manager to get valid tensors for initialization
                self.scene.update(dt=self.physics_dt)
```